articles/server-apis/java-spring-security/01-authentication.md

```diff
@@ -28,21 +28,21 @@ Save your changes.
 
 ![Configure JWT Signature Algorithm as RS256](/media/articles/server-apis/aspnet-core-webapi/jwt-signature-rs256.png)
 
-The next step is to configure your API to use the algorithm you chose. If you downloaded our seed, `HS256` is configured by default. To change this, edit the `/src/main/resources/auth0.properties` file as follows:
+The next step is to configure your API to use the algorithm you chose. If you downloaded our seed project, `HS256` is configured by default. To change this, edit the `/src/main/resources/auth0.properties` file as follows:
 
 ```properties
 #auth0.signingAlgorithm: HS256
 auth0.signingAlgorithm: RS256
 auth0.publicKeyPath: certificate/cert.pem
 ```
 
-The [library](https://github.com/auth0/auth0-spring-security-api) uses the `signingAlgorithm` property to determine which algorithm to use. The `publicKeyPath` attribute specifies the location of the certificate.
+The [library](https://github.com/auth0/auth0-spring-security-api) uses the `signingAlgorithm` attribute to determine which algorithm to use. The `publicKeyPath` attribute specifies the location of the certificate.
```

Comment:
@camimonse Why change this to `attribute` since the file is named `auth0.properties`?